sdk/core/azure-core/test/ut/operation_test.hpp

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include <azure/core/context.hpp>
+#include <azure/core/nullable.hpp>
+#include <azure/core/operation.hpp>
+#include <azure/core/operation_status.hpp>
+#include <azure/core/response.hpp>
+
+using namespace Azure::Core;
```

Comment:
Normally I would yell about a using-directive in a header, but since this is test code it's not a HUGE issue. I will point out that this particular using-directive is unnecessary: All the code in this header is defined in a subnamespace of `Azure::Core`, so unqualified name lookup will find things in `Azure::Core` anyway.